### PR TITLE
[release-4.14] OCPBUGS-21174: Bump golang.org/x/net to v0.18.0 

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -119,8 +119,3 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
-
-replace (
-	golang.org/x/net => golang.org/x/net v0.10.0
-	golang.org/x/sys => golang.org/x/sys v0.8.0
-)
```
Comment (davidvossel) [inline go.mod:126]: doesn't this need to be pinned to 0.18.0 here?

File: go.sum [hunk 2]
```diff
@@ -500,10 +500,50 @@ golang.org/x/mod v0.2.0/go.mod h1:s0Qsj1ACt9ePp/hMypM3fl4fZqREWJwdYDEqhRiZZUA=
...
+golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4/go.mod h1:p54w0d4576C0XHj96bSt6lcn1PtDYWL6XObtHCRCNQM=
+golang.org/x/net v0.0.0-20210421230115-4e50805a0758/go.mod h1:72T/g9IO56b78aLF+1Kcs5dz7/ng1VjMUvfKvpfy+jM=
+golang.org/x/net v0.0.0-20210428140749-89ef3d95e781/go.mod h1:OJAsFXCWl8Ukc7SiCT/9KSuxbyM7479/AVlXFRxuMCk=
+golang.org/x/net v0.0.0-20210525063256-abc453219eb5/go.mod h1:9nx3DQGgdP8bBQD5qxJ1jj9UTztislL4KSBs9R2vV5Y=
+golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d/go.mod h1:9nx3DQGgdP8bBQD5qxJ1jj9UTztislL4KSBs9R2vV5Y=
+golang.org/x/net v0.0.0-20210825183410-e898025ed96a/go.mod h1:9nx3DQGgdP8bBQD5qxJ1jj9UTztislL4KSBs9R2vV5Y=
+golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f/go.mod h1:9nx3DQGgdP8bBQD5qxJ1jj9UTztislL4KSBs9R2vV5Y=
+golang.org/x/net v0.0.0-20211209124913-491a49abca63/go.mod h1:9nx3DQGgdP8bBQD5qxJ1jj9UTztislL4KSBs9R2vV5Y=
+golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd/go.mod h1:CfG3xpIq0wQ8r1q4Su4UZFWDARRcnwPjda9FqA0JpMk=
+golang.org/x/net v0.0.0-20220225172249-27dd8689420f/go.mod h1:CfG3xpIq0wQ8r1q4Su4UZFWDARRcnwPjda9FqA0JpMk=
```
Comment (davidvossel) [inline go.sum:544]: what versions are these golang.ort/x/net dependencies?

File: go.mod (hunk 1 above)
Comment (qinqon) [reply go.mod:126]: Looks like after rebase everything is fine so maybe we don't need this PR 
```
golang.org/x/net v0.18.0 // indirect 
```

File: go.sum (hunk 2 above)
Comment (qinqon) [reply go.sum]: I have "pin" tro 0.18 so these ones disappear.

**Review by davidvossel — COMMENTED** [review]
/lgtm
/approve

**Comment by qinqon** [comment]
/override ci/prow/hypershift-kubevirt-conformance-baremetalds

**Pull request merged**